### ISSUE TITLE
Unknown model (vit_base_resnet50_384)

### ISSUE THREAD

**Issue by EkjotSingh898** [intel-isl/DPT#14]
Hi authors,
Thanks for your code,
While running run_monodepth.py , I got this error "Unknown model (vit_base_resnet50_384)". I'm using conda environment and installed timm using pip. and the version of timm I'm using is 0.4.5.

**Comment by Christoph9402** [comment]
Hi, I am having the same problem. Did you find a solution yet?

**Comment by Christoph9402** [comment]
Update: uninstalling timm and reinstalling with pip solved the issue for me. Now I have version 0.4.5 installed.

**Comment by ranftlr** [comment]
Yes, likely a version conflict. You could try installing into a fresh conda environment:

```shell
conda create --name dpt python=3.7
conda activate dpt
pip install torch torchvision opencv-python timm
```

**Comment by Hyyzhangrui** [comment]
as Christoph9402 said, update the timm's version to 0.4.5 ，good working！
